client/payment-details/payment-method/klarna/index.js

```diff
@@ -0,0 +1,146 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies.
+ */
+import PaymentDetailsPaymentMethodDetail from '../detail';
+
+/**
+ * Extracts and formats payment method details from a charge.
+ *
+ * @param {Object} charge The charge object.
+ * @return {Object}       A flat hash of all necessary values.
+ */
+const formatPaymentMethodDetails = ( charge ) => {
+	const { billing_details: billingDetails, payment_method: id } = charge;
+
+	const {
+		country: countryCode,
+		verified_name: verifiedName,
+		payment_method_category: paymentMethodCategory,
+		preferred_locale: preferredLocale,
+	} = charge.payment_method_details.klarna;
```

Comment:
Looking at Stripe [docs](https://stripe.com/docs/api/charges/object#charge_object-payment_method_details-klarna), the `country` and `verified_name` are not informed for Klarna so I believe we can remove them.